static/src/js/util/url/collectionsEncoders.js

```diff
@@ -201,6 +221,15 @@ const decodedSelectedVariables = (pgParam) => {
   return variableIds.split('!')
 }
 
+const decodedSwodlrDownload = (pgParam) => {
+  const { swod: swodlrData } = pgParam
+  if (!swodlrData) {
```

Comment:
Just for consistency with the other decoders, I might do `if (!swodlrData) return undefined`